doom-modeline-segments.el

```diff
@@ -2090,6 +2090,69 @@ mouse-3: Fetch notifications"
      (doom-modeline-spc))))
 
 
+;;
+;; Now Playing
+;;
+
+(defvar doom-modeline--now-playing-tokens '())
+(defun doom-modeline--now-playing-update ()
+  "Update the tokens for now-playing."
+  (when (and doom-modeline-now-playing
+             (require 'async nil t))
+    (async-start
+     `(lambda ()
+        (require 'subr-x)
+        ,(async-inject-variables
+          "\\`\\(load-path\\|auth-sources\\|doom-modeline-now-playing-format\\|doom-modeline-now-playing-ignored-players\\)\\'")
+        (string-trim (shell-command-to-string (format "playerctl metadata --ignore-player=%s --format '{{playerName}}|{{lc(status)}}|%s'" (mapconcat 'identity doom-modeline-now-playing-ignored-players ",") doom-modeline-now-playing-format))))
```

Comment:
Does it support all platforms?

Reply:
playerctl should work on any *NIX platform, I thought about making an adapter to making it more xplatform. If you would prefer me to do so I will :)

Reply:
yep, I'd like it works on all platforms, Windows, Linux, macOS.

Reply:
Roger that, I'll look into writing an adapter then